dockers/docker-router-advertiser/Dockerfile.j2

```diff
@@ -10,6 +10,9 @@ ENV DEBIAN_FRONTEND=noninteractive
 # Update apt's cache of available packages
 RUN apt-get update
 
+# Install radvd Debian package
+RUN apt-get -y install radvd
```

Comment:
just a thought, should we be always installing the latest or do we need to do some versioning?

Reply:
radvd doesn't seem to update frequently. However, it probably wouldn't hurt to add more versioning throughout SONiC so as to prevent surprises as pacakages get upgraded. I'll look into it here.

Reply:
Locked down the version in the latest commit.